src/main/java/com/robosh/ejournal/controller/AdminController.java

```diff
@@ -8,11 +8,7 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
u should set in your IDE import limit to 10

Reply:
file->settings->editor->code style->Java-> Imports tab-> Class count to use import with "*"